modules/ocf/manifests/firewall.pp

```diff
@@ -1,13 +1,15 @@
-class ocf::firewall {
+class ocf::firewall(
+  # Temporary variable for enabling mandatory input packet filtering on a
+  # host-by-host basis.
+  $allow_other_traffic = true,
+  ) {
   # Install prerequisite packages (that is, netfilter-persistent)
   include firewall
+  include ocf::firewall::pre
+  include ocf::firewall::post
 
-  include ocf::firewall::chains
+  require ocf::firewall::chains
```

Comment:
I'm not sure if this works the way it's supposed to. That is, I don't think it's necessarily the case that the `require` statement here creates a before/after relationship with the `include` statements above. What would certainly work is to maintain the `class` statement below and just remove the `require` attribute. This is the case because `class` statements behave like resources, which are indeed subject to the before/after relationship on the containing class imposed by the `require` statement.